content/20.pharmaceuticals.md

```diff
@@ -150,7 +150,20 @@ Overall, despite the study reporting clinical improvement in favipiravir-treated
 For example, despite the significant differences observed between the two treatment groups, follow-up analysis is necessary due to the small sample size.
 The selection of patients did not take into consideration important factors such as previous clinical conditions or sex, and there was no age categorization.
 The study was neither randomized nor blinded, and the baseline control group was another antiviral instead of a placebo.
-Therefore, randomized controlled trials are still required.
+Additionally, it should be noted that this study was temporarily retracted and then restored without an explanation [@url:https://retractionwatch.com/retracted-coronavirus-covid-19-papers/].
+
+In late 2020 and early 2021, the first randomized controlled trials of favipiravir for the treatment of COVID-19 released results [@doi:10.1016/j.ejps.2020.105631; @doi:10.1007/s00705-021-04956-9; @doi:10.1093/cid/ciaa1176].
+The first [@doi:10.1016/j.ejps.2020.105631] used a randomized, controlled, open-label design to compare two drugs, favipiravir and baloxavir marboxil, to SOC alone.
+Here, SOC included antivirals such as lopinavir/ritonavir and was administered to all patients.
+The primary endpoint analyzed was viral clearance at day 14.
+The sample size for this study was very small, with 29 total patients enrolled, and no significant effect of the treatments was found for the primary or any of the secondary outcomes analyzed, which included mortality.
+The second study was larger, with 96 patients enrolled, and also utilized a randomized design.
+This study enrolled only patients with mild to moderate symptoms and randomized them into two groups: one receiving CQ in addition to SOC, and the other receiving favipiravir in addition to SOC.
+This study reported a non-significant trend for patients receiving favipiravir to have a shorter hospital stay and less likelihood of progressing to mechanical ventilation or to an oxygen saturation < 90%.
+However, given the fact that favipiravir was being compared to CQ, which is now widely understood to be ineffective for treating COVID-19, these results do not suggest that favipiravir was likely to have had a strong effect on these outcomes. 
+On the other hand, another trial of 60 patients reported a significant effect of favipiravir on viral clearance at four days (a secondary endpoint analyzed) [@doi:10.1093/cid/ciaa1176].
```

Comment:
I think that you should definitely include that the effect was not observed at 10 days, especially as that was the primary endpoint.

Reply:
```suggestion
On the other hand, another trial of 60 patients reported a significant effect of favipiravir on viral clearance at four days (a secondary endpoint analyzed), but not at 10 days (the primary endpoint) [@doi:10.1093/cid/ciaa1176].
```